### ISSUE TITLE
数据集请求

### ISSUE THREAD

**Issue by wenchuanjiang** [FengheTan9/LLM4Seg#3]
尊敬的作者，您好，论文中涉及的数据集或者链接可以上传到github吗

**Comment by FengheTan9** [comment]
Thank you for your attention to our work. 
These datasets can be quickly downloaded from the official website (e.g. BUSI, ISIC 2018, and Kvasir).

**Comment by wenchuanjiang** [comment]
Dear authors,I have a few questions regarding the datasets used in your paper:
1. For the **TNSCUI** dataset, did you use the version released for the **MICCAI 2020 challenge**, or the one published in the paper *“An open access thyroid ultrasound image database”*?
2. Regarding the **ISIC 2018** dataset, did you only download the **training set (2,954 images)** from the official website for your experiments, or did you also include the **validation and test sets** and then perform your own data split?
3. For the **Kvasir** dataset, are you referring to the **Kvasir-SEG** dataset available on **Kaggle**? I noticed that the official Kvasir website only provides a dataset for **classification tasks** and does not include segmentation masks.
Thank you very much for your clarification!

**Comment by wenchuanjiang** [comment]
Hello, you mentioned in the paper that the Kvasir dataset was further split to create a test set for evaluation. How was this split performed?

**Comment by FengheTan9** [comment]
Please refer our [U-Bench](https://github.com/FengheTan9/U-Bench) for accessing datasets.
